etc/schema.json

```diff
@@ -3031,34 +3031,41 @@
             "optional": true,
             "properties": {
                 "cyu": {
+                    "description": "ja3-like fingerprint",
                     "type": "array",
                     "minItems": 1,
                     "items": {
                         "type": "object",
                         "properties": {
                             "hash": {
+                                "description": "hex representation",
```

Comment:
looking at https://jasonish-suricata.readthedocs.io/en/evedoc-v2/output/eve/eve-json-format.html?highlight=quic#output-reference, I think it would be better to have something like `cyu hash hex representation`.

Reply:
would also be good to mention this is limited to older QUIC variants (same for ua/sni)

Reply:
Same for UA, but SNI is also on standard version, extracted from the dedicated TLS client hello extension ;-)